app/src/components/market/common/currency_selector/index.tsx

```diff
@@ -66,22 +77,24 @@ export const CurrencySelector: React.FC<Props> = props => {
     currentItem = 0
   }
 
-  tokens.forEach(({ address, image, symbol }, index) => {
-    currencyDropdownData.push({
-      content: image ? <TokenItem image={image} text={symbol} /> : symbol,
-      extraContent: balance,
-      onClick: !disabled
-        ? () => {
-            onChange(address)
-          }
-        : () => {
-            return
-          },
+  tokens
+    .filter(({ address }) => filters.length === 0 || filters.indexOf(address.toLowerCase()) >= 0)
+    .forEach(({ address, image, symbol }, index) => {
+      currencyDropdownData.push({
+        content: image ? <TokenItem image={image} text={symbol} /> : symbol,
+        extraContent: balance,
+        onClick: !disabled
+          ? () => {
```

Comment:
could this ternary operator be done without repeating whole arrow function? onClick: ()=>{!disabled?onChange(address):return}

Reply:
```
() => {
          if (!disabled) onChange(address)
        }
```
I think this is enought